pulsar-functions/instance/src/main/python/python_instance_main.py

```diff
@@ -123,21 +123,21 @@ def main():
     # run pip install to target folder  deps folder
     zpfile = zipfile.ZipFile(str(args.py), 'r')
     zpfile.extractall(os.path.dirname(str(args.py)))
-    basename = os.path.splitext(str(args.py))[0]
+    basename = os.path.basename(os.path.splitext(str(args.py))[0])
```

Comment:
when specify `--py` with relative path like `py/exclamation.zip`, the `deps_dir` will become `py/py/exclamation/deps` while the correct one is `py/exclamation/deps`